### PR TITLE
fixes for auditwheel to depend on wheel 0.34

### PR DESCRIPTION
**Pull request by Czaki**
wheel in version 0.34 introduce fix for pypy tags https://github.com/pypa/wheel/issues/328

### CODE REVIEW

File: auditwheel/wheeltools.py [hunk 1]
```diff
@@ -27,6 +28,16 @@ class WheelToolsError(Exception):
     pass
 
 
+def open_for_csv(name, mode):
```
Comment (lkollar) [inline auditwheel/wheeltools.py]: Given that auditwheel only runs on Python 3 and there is only one place where this function is called, we could safely replace that call with `open(name, mode, newline='', encoding='utf-8')`. That avoids the unnecessary version check and makes the code more readable.

Comment (Czaki) [reply auditwheel/wheeltools.py]: I introduce this change. Could you restart failed steep. It fail on timeout error.

Comment (lkollar) [reply auditwheel/wheeltools.py]: Restarted the ppc64 build.

**Comment by Czaki** [comment]
@lkollar  When you think it will be next release?

**Pull request merged**